packages/terra-form/src/Input.jsx

```diff
@@ -37,34 +37,38 @@ const defaultProps = {
   value: undefined,
 };
 
-const Input = ({
-  defaultValue,
-  name,
-  required,
-  value,
-  ...customProps
-}) => {
-  const additionalInputProps = Object.assign({}, customProps);
+class Input extends React.Component {
+  render() {
+    const {
+      defaultValue,
+      name,
+      required,
+      value,
+      ...customProps
+    } = this.props;
+    const additionalInputProps = Object.assign({}, customProps);
 
-  if (required) {
-    additionalInputProps['aria-required'] = 'true';
-  }
+    if (required) {
+      additionalInputProps['aria-required'] = 'true';
+    }
 
-  if (value !== undefined) {
-    additionalInputProps.value = value;
-  } else {
-    additionalInputProps.defaultValue = defaultValue;
-  }
+    if (value !== undefined) {
+      additionalInputProps.value = value;
+    } else {
+      additionalInputProps.defaultValue = defaultValue;
+    }
 
-  return (
-    <input
-      name={name}
-      required={required}
-      {...additionalInputProps}
-      className={classNames('terra-Form-input', additionalInputProps.className)}
-    />
-  );
-};
+    return (
+      <input
+        name={name}
+        ref={(input) => { this.textInput = input; }}
```

Comment:
The only change made in this file is adding a ref on line 64 and changing Input from a functional component to a class component. The reason for this change is because the time-input needs a reference to the hour Input and minute Input so that it can move focus between them. The ref attribute only works on a class component and does not work on a functional because there is no instance of it. See https://facebook.github.io/react/docs/refs-and-the-dom.html#refs-and-functional-components